### ISSUE TITLE
import: Doesn't remove temporary files when download failed

### ISSUE THREAD

**Issue by mroutis** [iterative/dvc#1488]
*DVC version*: 0.23.2

`dvc import 'https://en.wikipedia.org/wiki/Main_Page'`

```
Importing 'https://en.wikipedia.org/wiki/Main_Page' -> '/home/mroutis/tmp/Main_Page'
[##############################] 100% Main_Page
Error: failed to import https://en.wikipedia.org/wiki/Main_Page - could not find an ETag for 'https://en.wikipedia.org/wiki/Main_Page'
```

`ls ~/tmp` -> `Main_Page`

---

`dvc import https://s3-us-west-2.amazonaws.com/dvc-share/get-started/data.xml`

```
Importing 'https://s3-us-west-2.amazonaws.com/dvc-share/get-started/data.xml' -> '/home/mroutis/tmp/data.xml'
[##                            ] 7% data.xml^CError: interrupted by the user

Having any troubles? Hit us up at https://dvc.org/support, we are always happy to help!
```

`ls ~/tmp` -> `data.xml.e5f6cf5a-a919-43bf-949b-dfd3754227f1`

**Comment by mroutis** [comment]
Although, it would be great if you could resume "importing" big files :thinking:

**Comment by mroutis** [comment]
Also, there's the oportunity to show a better message, like: `failed te import x... You can add it with dvc add` or something more acitonable

**Comment by efiop** [comment]
I don't think we should delete them though. tmp will be cleaned up on the next reboot(might differ on your distro), so it is not a big deal. And yes, when we will be able to resume downloads(which is not a tiny task, since the implementation is remote type specific), we would not want to delete these files as well.

**Comment by mroutis** [comment]
> tmp will be cleaned up on the next reboot(might differ on your distro), so it is not a big deal

The temporary files are created in the working directory, in this case, mine was `/home/mroutis/tmp`, sorry if it caused confusion :sweat_smile: 

> And yes, when we will be able to resume downloads(which is not a tiny task, since the implementation is remote type specific), we would not want to delete these files as well.

Should we re-open https://github.com/iterative/dvc/issues/108 and close this one, @efiop ?

**Comment by efiop** [comment]
Right, we are using <pre>fname + <font color="#CC0000">&apos;.&apos;</font> + <font color="#06989A">str</font>(uuid.uuid4())</pre> as a temp file name, so it is even more visible, since it places a not-hidden temp file with a meaningful name right in your working directory :)

Thanks for linking #108 ! :slightly_smiling_face: We used to have `--continue` support, but it got lost during the redesign :slightly_frowning_face:  Sure, let's reopen it.

@pared is working on improving the error message, so let's leave this issue opened for now.

**Comment by efiop** [comment]
This has been closed accidentally.

**Comment by efiop** [comment]
Fixed by #1501 . The `resume` part is re-opened #108
